components/ViroScene.tsx

```diff
@@ -174,7 +174,9 @@ export class ViroScene extends ViroBase<Props> {
   render() {
     // Uncomment this line to check for misnamed props
     //checkMisnamedProps("ViroScene", this.props);
-
+    console.log("[ViroScene].render");
+    console.log("[ViroScene].render", this);
+    console.log("[ViroScene].render", this.props);
```

Comment:
@robertjcolley is these `console.log` are necessary?

Reply:
Nope! I'll remove them in the next release I'm doing this week. Sorry about that! There are a few other places I think I put them.